app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= @mechanic.name %></h1>
+
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+
+<div id="mechanic_rides">
+  <h2>Current rides they're working on:</h2>
+  <% @mechanic.rides.each do |ride| %>
+    <p><%= ride.name %></p>
+  <% end %>
+</div>
+
+<div id="add_ride_to_mechanic">
+  <h2>Add a ride to workload:</h2>
+  <%= form_with url: mechanic_rides_path(@mechanic), data: {turbo: :false} do |f| %>
+    <%= f.label :ride_id, 'Ride Id:' %>
+    <%= f.number_field :ride_id %>
+    <%= f.submit "Submit" %>
+  <% end %>
+</div>
```

Comment:
Everything on the view looks good. Here's what `form_with model:` would look like:

```ruby
<%= form_with model: [@mechanic, @mechanic.rides.new], data: { turbo: :false } do |f| %>
  <%= f.label :ride_id, 'Ride Id:' %>
  <%= f.number_field :ride_id %>
  <%= f.submit "Submit" %>
<% end %>
```

This would produce a path like this: `"/mechanics/:id/rides"`.

Let's break `[@mechanic, @mechanic.rides.new]` down.
* `@mechanic` has an ID so the path is `"/mechanics/:id"`
* `@mechanic.rides` is a collection of all the mechanics rides, when we add `.new` we are saying that we want to add a new ride to it. Think about what we usually have in our `new` actions when we are doing sad paths, it' usually something like `@ride = Ride.new` or something like this. Here, this is the same thing, setting up a new ride to add to the list of mechanic rides, and at the end of the day, because we are using `.new`, this ride doesn't have an ID so the path is "/rides".
* If we add those 2 paths together, we get `"/mechanics/:id"` + `"/rides"` = `"/mechanics/:id/rides"`

Hope that helps.

Note: Instead of doing `@mechanic.rides.new` in the view, you would probably set this up in the controller action like `@ride = @mechanic.rides.new` and just do `[@mechanic, @ride]` in the view.